tex/chProofs.tex

```diff
@@ -4,6 +4,7 @@
 Set Implicit Arguments.
 Unset Strict Implicit.
 Unset Printing Implicit Defensive.
+% [DG] This looks outdated -- at least it doesn't compile.
```

Comment:
Good question. I remember pasting these lines into https://x80.org/collacoq/ and not being able to step through them. But if I try it now, everything works... Sorry then! Maybe it used to be a version conflict, or I just mispasted (Collacoq sometimes pastes things into a place where a Windows user wouldn't expect it).